_posts/archived/2005-05-30-thereisnoperfectdesign.aspx.markdown

```diff
@@ -8,7 +8,6 @@ categories:
 - code
 redirect_from:
   - /archive/2005/05/29/thereisnoperfectdesign.aspx
-  - /archive/2005/05/30/thereisnoperfectdesign.aspx
```

Comment:
@parkr mind opening up an issue in jekyll redirect from to guard against this? Glad to get a PR together.